modules/alloy/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## Unreleased
 
+### Added
+- Added `onToggled` event for `SplitFloatingToolbar` and `SplitSlidingToolbar` components. #TINY-9271
```

Comment:
```suggestion
- Added the `onToggled` callback for `SplitFloatingToolbar` and `SplitSlidingToolbar` components. #TINY-9271
```
Just correcting the terminology here is no event here that's fired, it's a callback.

Reply:
Updated the changelog [here](https://github.com/tinymce/tinymce/pull/8238/commits/2d3044613f2f55c8065be436df1a284376972b91)